index.bs

```diff
@@ -867,6 +880,25 @@ spec: secure-contexts; urlPrefix: https://www.w3.org/TR/secure-contexts/
               source has side effects such as enabling different encoding
               modules.
             </li>
+            <li>
+              If the user agent is able to encode the media represented by
+              <var>configuration</var> and <var>configuration</var> has a type
+              of {{MediaEncodingType/webrtc}}, run the following substeps:
+              <ol>
+                <li>Let <var>webrtc</var> be a
+                {{WebRTCMediaCapabilitiesInfo}} dictionary.</li>
+                <li>If audio is represented by <var>configuration</var>, set
```

Comment:
Similarly, can we link to something that explicitly determines "audio" and "video" types are being represented?